docs/cinder-csi-plugin/features.md

```diff
@@ -13,6 +13,7 @@
   - [Volume Cloning](#volume-cloning)
   - [Multi-Attach Volumes](#multi-attach-volumes)
   - [Liveness probe](#liveness-probe)
+  - [Set file type in provisioner](#set-file-type-in-provisioner)
```

Comment:
This doesnt seem to be the right place. How about we add seperate heading "Sidecar Compatibility" before Supported Features here , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md#supported-features . 

And add the current supported versions of sidecars and add this as note